src/redis_lock/django_cache.py

```diff
@@ -19,6 +20,36 @@ def __client(self):
     def lock(self, key, expire=None, id=None):
         return Lock(self.__client, key, expire=expire, id=id)
 
+    def get_or_set_locked(self, key, value_creator, version=None,
+                          expire=None, id=None, lock_key=None,
+                          timeout=DEFAULT_TIMEOUT):
+        """
+        Fetch a given key from the cache. If the key does not exist, the key is added and
+        set to the value returned when calling `value_creator`. The creator function
+        is invoked inside of a lock.
+        """
+        if lock_key is None:
+            lock_key = '{0}.set-lock'.format(key)
```

Comment:
Could we use perhaps a different scheme here? In the [same vein](https://github.com/ionelmc/python-redis-lock/blob/master/src/redis_lock/__init__.py#L173-L174) as the normal key naming. Eg: `lock_key = 'get_or_set:{0}'.fomat(key)`

That would allow people to easily find all the keys related to this feature.